deployments/container/Dockerfile.ubi8

```diff
@@ -14,23 +14,25 @@
 
 ARG GOLANG_VERSION=1.20.5
 ARG CUDA_IMAGE=cuda
-ARG CUDA_VERSION=12.2.0
+ARG CUDA_VERSION=12.3.0
 ARG BASE_DIST=ubi8
 FROM nvidia/cuda:${CUDA_VERSION}-base-${BASE_DIST} as build
 
+ARG GOLANG_VERSION=1.20.5
+
 RUN yum install -y \
     wget make git gcc \
-     && \
+    && \
     rm -rf /var/cache/yum/*
 
 RUN set -eux; \
     \
     arch="$(uname -m)"; \
     case "${arch##*-}" in \
-        x86_64 | amd64) ARCH='amd64' ;; \
-        ppc64el | ppc64le) ARCH='ppc64le' ;; \
-        aarch64) ARCH='arm64' ;; \
-        *) echo "unsupported architecture" ; exit 1 ;; \
+    x86_64 | amd64) ARCH='amd64' ;; \
```

Comment:
Why the change?

Reply:
vscode indentation when using the docker extension (if runs a lint checker on Dockerfiles)